libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/infopanel/InfoPanelBehavior.kt

```diff
@@ -5,10 +5,17 @@ import kotlinx.coroutines.flow.asStateFlow
 
 internal class InfoPanelBehavior {
 
-    private val _infoPanelBehavior = MutableStateFlow<Int?>(null)
-    val infoPanelBehavior = _infoPanelBehavior.asStateFlow()
+    private val _bottomSheetState = MutableStateFlow<Int?>(null)
+    val bottomSheetState = _bottomSheetState.asStateFlow()
 
-    fun updateBehavior(newState: Int) {
-        _infoPanelBehavior.value = newState
+    private val _slideOffset = MutableStateFlow<Float>(-1f)
+    val slideOffset = _slideOffset.asStateFlow()
+
+    fun updateBottomSheetState(newState: Int) {
+        _bottomSheetState.value = newState
+    }
+
+    fun updateSlideOffset(slideOffset: Float) {
+        _slideOffset.value = slideOffset.coerceIn(-1f, 1f)
```

Comment:
Since you are willing to fix the values yourself, could you also replace NaNs with 0?